streams.json

```diff
@@ -757,121 +1004,152 @@
+          "tag": "4.3.10.Final",
+          "group_id": "org.hibernate",
+          "artifact_id": "hibernate-project",
+          "version": "4.3.10.Final",
+          "gav": "hibernate-project:org.hibernate:4.3.10.Final",
           "comment": ""
         },
         {
           "component_name": "hibernate-hibernate-jpa-api",
           "contacts": [
               "ebernard@redhat.com",
-              "gbadner@redhat.com"
+            "gbadner@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/hibernate/hibernate-jpa-api",
+          "repository_url": "https://github.com/hibernate/hibernate-jpa-api.git",
           "codebase": "master",
-          "tag": "2.10.Final",
+          "tag": "",
+          "group_id": "org.hibernate.javax.persistence",
+          "artifact_id": "hibernate-jpa-api-project",
           "version": "1.0.0.Final",
-          "gav": "org.hibernate.javax.persistence",
-          "comment": "NOTE: maintenance is nightmare, gradle vs maven, in maintenance master API version is tag name? Artifacts in EAP 1.0.0.Final-redhat-2"
+          "gav": "hibernate-jpa-api-project:org.hibernate.javax.persistence:1.0.0.Final",
+          "comment": "TAG not found. NOTE: maintenance is nightmare, gradle vs maven, in maintenance master API version is tag name? Artifacts in EAP 1.0.0.Final-redhat-2"
         },
         {
           "component_name": "hibernate-hibernate-search",
           "contacts": [
               "ebernard@redhat.com",
-              "gbadner@redhat.com"
+            "gbadner@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/hibernate/hibernate-search",
+          "repository_url": "https://github.com/hibernate/hibernate-search.git",
           "codebase": "5.5",
           "tag": "5.5.8.Final",
+          "group_id": "org.hibernate",
+          "artifact_id": "hibernate-search-parent",
           "version": "5.5.8.Final",
-          "gav": "org.hibernate",
+          "gav": "hibernate-search-parent:org.hibernate:5.5.8.Final",
           "comment": ""
         },
         {
           "component_name": "hibernate-hibernate-validator",
           "contacts": [
               "ebernard@redhat.com",
-              "gbadner@redhat.com"
+            "gbadner@redhat.com"
           ],
           "repository_type": "GIT",
-          "repository_url": "https://github.com/hibernate/hibernate-validator",
+          "repository_url": "https://github.com/hibernate/hibernate-validator.git",
           "codebase": "5.3",
           "tag": "5.3.5.Final",
+          "group_id": "org.hibernate",
+          "artifact_id": "hibernate-validator-parent",
           "version": "5.3.5.Final",
-          "gav": "org.hibernate",
+          "gav": "hibernate-validator-parent:org.hibernate:5.3.5.Final",
           "comment": ""
         },
         {
           "component_name": "hibernate-jpa-api",
```

Comment:
There is already a hibernate-hibernate-jpa-api component which seems to be the same as this one, I suggest leaving this one out from the final file.

Reply:
hibernate-hibernate-jpa-api and hibernate-hibernate-validator ? 
they seem to be different, versions and repositories.

Reply:
OK, the second svn repository seems redundant.

Reply:
I will remove hibernate-jpa-api then because hibernate-hibernate-jpa-api is in prod KB.